test/docker-env/grafana/set_admin_password.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ "x${GRAFANA_PASSWORD}" != "x" ]; then
```

Comment:
The default is possible now, if the environment variable is not there, it will be the default. Just grafana will ask to change it (there is a skip button). I will look into if we can have a passwordless grafana for development purposes.